src/App.jsx

```diff
@@ -1,27 +1,47 @@
-import logo from './logo.svg';
 import './App.css';
+import React, {useState} from "react";
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-        <h1>Luis esta aqui</h1>
-        <h2>Canca se fue de viaje</h2>
-      </header>
+    <div className="container">
+      <TopBar/>
+      <Button/>
     </div>
   );
 }
 
 export default App;
+
+function TopBar(){
```

Comment:
TopBar should have its own file otherwise if you keep adding components in this file it will be hundreds of lines of code and we never want that

Reply:
ok auntie
fixed it